view.react/src/api/services/session.service.js

```diff
@@ -1,36 +1,28 @@
 import {USER_DATA_KEY} from "../../constants/session.keystorage";
-import {sendPostRequest} from "../../helpers/RequestHelper";
 
-const URL = "/api/v2/session";
 
-export const login = (body) => {
-    return sendPostRequest(URL, body)
-        .then(data => {
-            if (data.token) {
-                localStorage.setItem(USER_DATA_KEY, JSON.stringify(data));
-            }
-            return data;
-        })
-}
-
-export const logout = () => {
+export const logout = (instance) => {
     localStorage.removeItem(USER_DATA_KEY);
+    instance.logoutRedirect().catch(e => {
+        console.error(e);
+    });
 }
 
 export const getCurrentUser = () => {
-    const user = JSON.parse(localStorage.getItem(USER_DATA_KEY)) || {userRoles: []}
+    const user = JSON.parse(localStorage.getItem(USER_DATA_KEY)) || {roles: []}
     return {
         ...user,
-        isLeader: user.userRoles.includes("ROLES_LEADER"),
-        isAdmin: user.userRoles.includes("ROLES_ADMIN")
-    }
+        isLeader: user?.roles.includes("ROLES_LEADER"),
+        isAdmin: user?.roles.includes("ROLES_ADMIN")
+    };
 }
 
 export const getFullUserName = () => {
     const user = getCurrentUser();
-    return `${user.name} ${user.surname}`;
+    return user.name ? `${user?.name} ${user?.surname}` : '';
 }
 
 export const getUserTeams = () => {
-    return getCurrentUser().teams;
+    const user = getCurrentUser();
+    return user.teams? user.teams : [];
```

Comment:
You can use nullish coalescing operator 
```suggestion
    return user.teams ?? [];
```